docs/installation.md

```diff
@@ -0,0 +1,76 @@
+
+If you are not using the dev container you will need to install the dependencies in the requirements.txt file. Using a python virtual environment is recommended but it is not necessary. If you want to run the python linters and tests you will also need the dependencies in the requirements-test.txt file. The JavaScript and accessibility test requirements are covered in the [testing section](testing.md).
+
+To set up the virtual environment and install the python requirements use the following commands.
+
+```bash
+python3 -m venv venv
+source venv/bin/activate
+pip3 install -r requirements.txt
+```
+
+If you are using windows then you may need to change the way the that you activate the virtual environment (the second line in the example above) and instead use one of the following two lines of code depending on the type of shell you are using.
+
+For CMD:
+
+```ps
+.\venv\Scripts\activate.bat
+```
+
+For Power shell:
+
+```ps
+.\venv\Scripts\activate.ps1
+```
+
+Flask provides a development webserver which is good enough to evaluate the software and for local testing/development. For use in production follow the advice provided in the [Flask documentation](https://flask.palletsprojects.com/en/3.0.x/deploying/).
+
+## Running the provided examples
+
+This sequence of commands will allow you setup and run one of the pre-configured examples. Examples are provided for each of the three types of item pair selection described in the introduction, the example files can be found in the `examples` directory inside the `comparison_interface` directory.
+
+1. equal item weight options
+    + config-equal-item-weights.json
+    + config-equal-item-weights-preference.json
+1. config-custom-item-weights.json
+
+### Initial setup
+
+If you are not using the dev container provided then:
+
++ rename the `example.images` file in `comparison_interface/static/` to `images`.
++ rename the `example.flask.py` file in `comparison_interface/configuration/` to `flask.py`.
+
+When running in production the secret key in the flask.py should also be changed.
+
+Open a terminal and run these commands replacing ```[configuration_file_name]``` with the name of the configuration file you want to try.
+
+```bash
+flask --debug setup examples/[configuration_file_name]
+flask --debug run --port=5001
+```
+
+In your browser navigate to <http://localhost:5001>
+
+You should see the registration page of the website and all of the functions should be working so you can test the features with a small set of images.
+
+Any free port number can be used to host the website by changing the port number in the run command. 5001 is used in all of the examples because it is the port number that is open in the dev container setup mentioned above.
```

Comment:
Should we recommend a range of free port numbers, e.g. 1024 - 65535?

Reply:
maybe, I don't really understand port numbers - I rarely stray out of the 8000's. I inherited the 5000 ones in this code. Is that the range you would recommend?